### PR TITLE
API changes

### PR DESCRIPTION
**Pull request by shilch**
Made the API more RESTful and corrected some errors. Instead of prefixing every endpoint with /v1, I wrote one sentence that notes that all endpoints have to be prefixed with it.

### CODE REVIEW

File: CAMP.md [hunk 1]
```diff
@@ -4,90 +4,112 @@ CAMP is a sub-protocol of OpenCap. CAMP is an optional protocol that allows a se
...
+Content-Type: application/json
+
 {
-    "jwt": "ujahsdbf9dfsf.sdf8ihnsd9fgasdsdfusd8f0adf78f.ud8fs0fbsf"
+    "jwt": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6ImFsaWNlIiwiZG9tYWluIjoiZG9tYWluLnRsZCIsImlhdCI6MTUxNjIzOTAyMn0.Kxy-elSGuiSzBv2s6JlqbFU3kxgOD-sg1fm7AgrRFDE"
 }
 ```
 
-## DELETE /v1/users
+## DELETE /domains/{domain}/users/{username}
```
[the remark this answers is not in the document]
Comment (shilch) [reply CAMP.md]: That's how RESTful APIs are designed. A delete is operated on an item (the username in this case) and not a resource set (users). I know that it sounds tautological but REST APIs look like this.

File: CAMP.md [hunk 2]
```diff
@@ -4,90 +4,112 @@ CAMP is a sub-protocol of OpenCap. CAMP is an optional protocol that allows a se
...
-```javascript
-Status: 200
-{}
+```
+HTTP/1.1 200 OK
+
 ```
 
-## PUT /v1/address
+## PUT /domains/{domain}/users/{username}/ledgers/{ledger_id}
```
[the remark this answers is not in the document]
Comment (shilch) [reply CAMP.md]: 'Asset' suggests that every asset (= every token on top of Ethereum, NEO, ...) gets its own ID which I don't think was the plan.

File: CAMP.md [hunk 3]
```diff
@@ -4,90 +4,112 @@ CAMP is a sub-protocol of OpenCap. CAMP is an optional protocol that allows a se
...
+PUT /domains/domain.tld/users/alice/ledgers/1 HTTP/1.1
+Authorization: earer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6ImFsaWNlIiwiZG9tYWluIjoiZG9tYWluLnRsZCIsImlhdCI6MTUxNjIzOTAyMn0.Kxy-elSGuiSzBv2s6JlqbFU3kxgOD-sg1fm7AgrRFDE
 
-Body:
-```javascript
 {
-    "asset": 0, // 0 = BTC
-    "type": "segwit",
-    "address": "bc1qvw0ytfntx6zs0lfsruem6xwj0mewng523ktatp"
+    "asset": 1
```
[the remark this answers is not in the document]
Comment (shilch) [reply CAMP.md]: Yeah, that's redundant. I wanted to delete the field.

File: README.md [hunk 4]
```diff
@@ -60,7 +60,8 @@ Using email address format does neither mean that a OpenCAP service has to be ru
 
 OpenCAP is the generic name to describe the entire alias protocol. There are three sub-protocols that allow servers to decide to what level they wish to provide alias hosting.
 
-They sub-protocol are listed in descending levels of importance. For example, all CAMP servers also must be CAP servers. All CAPP servers must also be CAMP and CAP servers.
+They sub-protocol are listed in descending levels of importance. For example, all CAMP servers also must be CAP servers. All CAPP servers must also be CAMP and CAP servers.
```
[the remark this answers is not in the document]
Comment (shilch) [reply README.md]: Okay

**Comment by lane-c-wagner** [comment]
What I mean is that if we want to change the name from "asset" (which I do)
it shouldn't be in this PR, that would be a separate PR.

**Comment by shilch** [comment]
I changed 'ledger' to 'asset' so we can correct that in another PR.

File: CAMP.md [hunk 5]
```diff
@@ -4,90 +4,112 @@ CAMP is a sub-protocol of OpenCap. CAMP is an optional protocol that allows a se
...
-Status: 200
-{}
 ```
+HTTP/1.1 200 OK
 
-## POST /v1/auth
+```
 
-Authorization Bearer {jwt}
+
```
Comment (lane-c-wagner) [inline CAMP.md]: Authorization is traditionally done via a POST. I think it makes more sense the way it was before with going to v1/auth and sending the data as json. Again, this is just more industry standard for authentication. your JWT example is better though see we can keep that

File: CAMP.md [hunk 6]
```diff
@@ -4,90 +4,112 @@ CAMP is a sub-protocol of OpenCap. CAMP is an optional protocol that allows a se
...
+PUT /domains/domain.tld/users/alice/assets/1 HTTP/1.1
+Authorization: earer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6ImFsaWNlIiwiZG9tYWluIjoiZG9tYWluLnRsZCIsImlhdCI6MTUxNjIzOTAyMn0.Kxy-elSGuiSzBv2s6JlqbFU3kxgOD-sg1fm7AgrRFDE
 
-Body:
-```javascript
 {
-    "asset": 0, // 0 = BTC
-    "type": "segwit",
-    "address": "bc1qvw0ytfntx6zs0lfsruem6xwj0mewng523ktatp"
+    "asset": 1
```
Comment (lane-c-wagner) [inline CAMP.md]: Still need to remove "asset" from the request body

File: CAMP.md (hunk 5 above)
Comment (lane-c-wagner) [reply CAMP.md]: Another reason why I don't like this is the path is inconsistent, the rest of our endpoints are /domains/users/assets this one randomly changes to /domains/users/tokens

**Pull request merged**